docs/java-rest/high-level/apis/search.asciidoc

```diff
@@ -178,3 +189,51 @@ cases need to be case accordingly:
 --------------------------------------------------
 include-tagged::{doc-tests}/SearchDocumentationIT.java[search-hits-singleHit-source]
 --------------------------------------------------
+
+==== Accessing Aggregations
```

Comment:
I wasn't sure if this should be a separate part after explaining the rest of the SearchResponse or better be grouped together with the part above about adding the aggregations to the request. In that case maybe both parts could got together and we should put everything under an "Aggregation" header after the general part about search?